packages/cdktf-cli/src/bin/cmds/handlers.ts

```diff
@@ -82,11 +85,20 @@ async function getProviderRequirements(provider: string[]) {
   return [...provider, ...providersFromConfig];
 }
 
-export async function convert({ language, provider, stack }: any) {
+export async function convert({
+  language,
+  provider,
+  stack,
+  withoutProject,
+}: any) {
   await initializErrorReporting();
   await displayVersionMessage();
 
+  const pkg = readPackageJson();
+
+  const tempDir = await fs.mkdtemp(path.join(os.tmpdir(), "cdktf-convert-"));
```

Comment:
Nit: we could move this into the if (useProject) to not create empty dirs by default